src/domain/messages/entities/schemas/message.schema.ts

```diff
@@ -1,57 +1,27 @@
-import { buildPageSchema } from '@/domain/entities/schemas/page.schema.factory';
-import { Schema } from 'ajv';
+import { buildZodPageSchema } from '@/domain/entities/schemas/page.schema.factory';
+import { SignatureType } from '@/domain/messages/entities/message-confirmation.entity';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
+import { z } from 'zod';
 
-export const MESSAGE_CONFIRMATION_SCHEMA_ID =
-  'https://safe-client.safe.global/schemas/messages/message-confirmation.json';
+export const MessageConfirmationSchema = z.object({
+  created: z.coerce.date(),
+  modified: z.coerce.date(),
+  owner: AddressSchema,
+  signature: HexSchema,
+  signatureType: z.nativeEnum(SignatureType),
```

Comment:
I didn't add a fallback here as we control this type and would otherwise need to extend `SignatureType` with `UNKNOWN`, for example. Would appreciate your thoughts.

Reply:
I think it's OK to not have a fallback value, as you said we own this type internally and it haven't changed for years. And if it does and we don't notice, we should be able to catch the related errors in development environments.